drizzlepac/updatehdr.py

```diff
@@ -292,9 +289,9 @@ def linearize(wcsim, wcsima, wcsref, imcrpix, f, shift, hx=1.0, hy=1.0):
         wcsref.wcs_pix2world(p.astype(np.float64), 1), 1).astype(np.longdouble)
 
     # derivative with regard to x:
-    u1 = ((p[1] - p[4]) + 8 * (p[3] - p[2])) / (6*hx)
+    u1 = ((p[1] - p[4]) + 8 * (p[3] - p[2])) / (6 * hx)
     # derivative with regard to y:
-    u2 = ((p[5] - p[8]) + 8 * (p[7] - p[6])) / (6*hy)
+    u2 = ((p[5] - p[8]) + 8 * (p[7] - p[6])) / (6 * hy)
 
```

Comment:
This can wait, but we should work to remove magic/hard-coded numbers and indices from the code.